### ISSUE TITLE
Parse errors: bad token/bad expression

### ISSUE THREAD

**Issue by TheSin-** [dak180/FreeNAS-Report#20]
These seem to be sed/awk errors, and I noticed on the report all my SSD drives are missing a value in the report (Load Unload Cycles) as well as I see this on each one

```
scsiPrintBackgroundResults Failed [scsi response fails sanity test]
```

I haven't dug in to find the line yet, thought I'd make a report first.

This repeats 8 times, which is how many Micron_5 SSD drives I have not including the 1 WDC SSD for a pool log

```
 ./report.sh -c report.cfg

Parse error: bad token
    <stdin>:1


Parse error: bad expression
    <stdin>:1


Parse error: bad expression
    <stdin>:1


Parse error: bad expression
    <stdin>:1
```

on a separate note, my boot flash drive on my TrueNAS X10 gives me a critical temp (40C) maybe that one needs to be able to run hotter then the others?

**Comment by dak180** [comment]
Run `./report.sh -dc report.cfg` and send me the dump file it emails you; I will have to take a look at what those drives output for smart data.

**Comment by dak180** [comment]
@TheSin- just to make sure, self tests are set to run; both short and long?

**Comment by dak180** [comment]
> on a separate note, my boot flash drive on my TrueNAS X10 gives me a critical temp (40C) maybe that one needs to be able to run hotter then the others?

SSDs in general can have a separate temp warning level set for them from HDDs since they generally have different characteristics. Depending on the SSD 40C may not be unreasonable.

**Comment by dak180** [comment]
Fixed in e86c4a55966b4dec7e0e3a6af210ac893d5d6108.
